.isort.cfg

```diff
@@ -3,4 +3,4 @@ use_parentheses=true
 multi_line_output=3
 include_trailing_comma=true
 line_length=79
-known_third_party = flag_engine,flask,pytest,requests,requests_futures,responses,urllib3
+known_third_party = flag_engine,pytest,pytest_mock,pytz,requests,requests_futures,responses,sseclient,urllib3
```

Comment:
This is sort of a strange inclusion into this PR. Why wasn't `pytest_mock`, for example, known as a third party prior?